org.bridgedb.uri.api/test/org.bridgedb.statistics/DataSetInfoTest.java

```diff
@@ -0,0 +1,42 @@
+package org.bridgedb.statistics;
+
+import org.junit.Assert;
+
+public class DataSetInfoTest {
+
+    @org.junit.Test
```

Comment:
This is just a note, but we should upgrade in a later patch to JUnit5, see https://github.com/bridgedb/BridgeDb/issues/115

Reply:
Surely, I'll start writing tests in JUnit5 and upgrade the older ones to JUnit5